### DISCUSSION TITLE
I can't find a block for 3 months.

### DISCUSSION THREAD

**Discussion by zecasa** [Chia-Network/chia-blockchain#15876] [Support]
This is complete bullshit. I was finding at least 2 blocks every month for 2 years. There is definitely a problem. I will never join a pool. 
I will not enter. I'm pretty sure the system is no longer working properly. Compressed plots causing the problem.
Someone should put a stop to this.

**Comment by heget** [comment]
Hi, I have the same hunch that something has changed and pools are preferred over solos. 
I have a fairly large solo farm.

**Reply by BrandtH22** [reply]
Keep in mind that the blockchain is unaware as to whether your plots are setup for solo or pool plotting so there is no preference for pools over solo.

Using compressed plots will give users more farmable space but they also require more energy to farm so there is a tradeoff.

Feel free to jump into our discord server and we can further review the logs and configuration to see if there are some optimizations that might assist with farming:
https://discord.gg/chia

**Reply by zecasa** [reply]
I joined but no one

**Reply by heget** [reply]
Give it time. It will definitely come.

**Comment by heget** [comment]
I will also add that when I join the pool I find blocks immediately within a day. Interesting isn't it?!

**Comment by BrandtH22** [comment]
Hey @zecasa , feel free to jump into the support channel in our discord server and I will be able to help review the system configuration and logs to determine where the issue might reside:
https://discord.gg/chia

**Reply by zecasa** [reply]
I joined your discord channel https://discord.gg/chia. My nick name is zecasa. Find me discord.

**Reply by BrandtH22** [reply]
Hey @zecasa, I was unfortunately OOO this weekend but will be available throughout the week to assist.
I have just replied to one of your comments in discord after thoroughly reviewing the threads.

Let's keep up the support in the discord server and we can update this ticket as we identify bugs and/or resolutions 👍
